clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/MetadataRepoBasedStaleTopicCheckerImpl.java

```diff
@@ -28,8 +28,8 @@ public boolean shouldTopicExist(String topic) {
           return false;
         }
       } else if (Version.isRealTimeTopic(topic)) {
-        if (!store.isHybrid()) {
-          LOGGER.warn("Store {} is not hybrid currently, but found real-time topic {}", storeName, topic);
+        if (!store.isHybrid() && !Version.containsHybridVersion(store.getVersions())) {
```

Comment:
It's possible that store is converted to batch but still have hybrid version. So checking for only hybrid versions is enough
```java
if (!Version.containsHybridVersion(store.getVersions()))
```